lib/ControlledVocab/tests/ControlledVocab-test.js

```diff
@@ -160,6 +164,22 @@ describe('ControlledVocab', () => {
       expect(cv.cancelButton).to.exist;
     });
 
+    describe('Cancel button is clicked', () => {
+      beforeEach(async () => {
+        await cv.cancelButton().click();
+      });
+
+      it('new button is enabled', () => {
```

Comment:
Please rename it to
```suggestion
      it('should enable a New button', () => {
```

Please check other places.